NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="dev" value="https://fiskaltrust.pkgs.visualstudio.com/_packaging/dev/nuget/v3/index.json" />
+  </packageSources>
```

Comment:
Should we really pull in a dependency to our internal feed?

Reply:
Good catch. We will need to wait then until the interface 1.3.1-rc2 is on NuGet.org